app/src/foss/java/com/google/android/gms/maps/MapView.java

```diff
@@ -53,8 +58,14 @@ public void getMapAsync(OnMapReadyCallback callback) {
 
   private void setDefaultConfiguration(Context context) {
     String userAgent = String.format("%s/%s", context.getString(R.string.app_name), BuildConfig.VERSION_NAME);
-    Configuration.getInstance().load(context, PreferenceManager.getDefaultSharedPreferences(context));
-    Configuration.getInstance().setUserAgentValue(userAgent);
+    final IConfigurationProvider config = Configuration.getInstance();
+    final NetworkManager         nm     = ApplicationDependencies.getNetworkManager();
+    config.setOsmdroidTileCache(new File(context.getCacheDir(), "tiles"));
+    config.load(context, PreferenceManager.getDefaultSharedPreferences(context));
```

Comment:
Another vestige of the original PR. Molly doesn't persist any preference value for OSM. We can safety remove this call to `load()` and rely on the default values provided by the library.